packages/components/src/Button/Button.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import { Box, system } from '@marigold/system';
 
-type ButtonProps = {};
+type ButtonProps = {
+  disabled?: boolean;
```

Comment:
isn't that part of the `ButtonAttributes` anyway? Do we have to add this ourselves 🤔

Reply:
true, we don't need it